internal/pkg/mapper/mapper_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package mapper_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestLog(t *testing.T) {
```

Comment:
`TestMapper`?

Reply:
Feel free to not use this BDD framework btw (per Chris' suggestion).

Reply:
The framework works great due to the `DescribeTable` feature!! It turned out extremely easy to write same test case with multiple inputs. In usual `testing` library in go, one has to do more boilerplate.

Reply:
The golang boilerplate looks like this https://github.com/google/cel-go/blob/master/interpreter/interpreter_test.go#L764-L776